### ISSUE TITLE
[Bug]: Shows "$title"

### ISSUE THREAD

**Issue by ghost** [gautamkrishnar/blog-post-workflow#49]
**Describe the bug**

I changed my blog-posts template to `$newline - $randomEmoji(💯,🔥,💫,🚀,🌮) [$title]($url)` and in the last article it shows
`$title <TITLE AND LINK>` [here](https://github.com/dart-coder) you can see it.

**Expected behavior**

I expected that it will show smth like this:
- 💯 [COOL ARTICLE](https://dev.to/cool-article)

**Screenshots**

https://user-images.githubusercontent.com/70339507/102758380-ba482880-4383-11eb-8a05-9bcaf954e4cb.png

**Workflow Yml Used**
```yml
name: Latest blog post workflow
on:
  schedule: # Run workflow automatically
    - cron: '0 * * * *' # Runs every hour, on the hour
  workflow_dispatch: # Run workflow manually (without waiting for the cron to be called), through the Github Actions Workflow page directly
jobs:
  update-readme-with-blog:
    name: Update this repo's README with latest blog posts
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: gautamkrishnar/blog-post-workflow@master
        with:
          feed_list: "https://dev.to/feed/dartcoder"
          template: "$newline - $randomEmoji(💯,🔥,💫,🚀,🌮) [$title]($url)"
```

**Additional context**

Thanks for helping.

**Comment by gautamkrishnar** [comment]
@dart-coder hope it's fixed right now. I am unable to see the issue on the URL you provided. Please close this one if its working fine.

**Comment by qWici** [comment]
@gautamkrishnar nope. It's bugged. You can check this out on [profile repo](https://github.com/qWici/qWici).

**Comment by gautamkrishnar** [comment]
@qWici looks like it is working fine.

**Comment by gautamkrishnar** [comment]
oh yea got it $title is there.

**Comment by gautamkrishnar** [comment]
@qWici fix for this is released on version 1.5.1 https://github.com/gautamkrishnar/blog-post-workflow/releases/tag/1.5.1

**Comment by qWici** [comment]
@gautamkrishnar cool, thanks)
